packages/geo/src/lib/style/style-modal/layer/style-modal-layer.component.html

```diff
@@ -14,6 +14,7 @@ <h2 mat-dialog-title class="mat-typography">
         </span>
         <igo-color-picker-form-field
           formControlName="fill"
+          (colorChange)="setLayerFillColor($event)"
```

Comment:
On pourrait avoir les changements de valeurs ainsi dans le Typescript de la composante:
```suggestion
    this.form
      .get('fill')
      .valueChanges.subscribe((color) => this.handleFillColor(color));
```

Reply:
Il faut cliquer sur  Appliquer/OK pour récupérer la couleur

Reply:
![chrome_2023-11-20_16-24-49](https://github.com/infra-geo-ouverte/igo2-lib/assets/7397743/db01dbb0-ec20-47e0-8116-f6afe2e13016)

Reply:
Vraiment plus convivial pour choisir une couleur, surtout si des superpositions/conflits visuel de couleur sont possible.